jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -14,7 +14,7 @@ pipeline {
     parameters {
         string(
             name: 'COMPONENT_NAME',
-            description: 'If this field contains a component name (e.g. OpenSearch-Dashboards), will build with "--component <COMPONENT_NAME>", else build everything in the INPUT_MANIFEST.',
+            description: 'If this field contains one or more component names (e.g. OpenSearch), will build with "--component <COMPONENT_NAME> ...", else build everything in the INPUT_MANIFEST.',
```

Comment:
Can we change the `e.g.` section to be `(e.g. OpenSearch-Dashboards reportsDashboards ...)`